delta/extensions/metrics.py

```diff
@@ -71,7 +71,6 @@ def __init__(self, label=None, class_id: int=None, name: str=None, binary: int=F
         self._total_class = self.add_weight('total_class', initializer='zeros')
         self._true_positives = self.add_weight('true_positives', initializer='zeros')
 
-    # sample_weight is unused but required by tensorflow
```

Comment:
can you put back this comment? Added this because we recently deleted sample_weight and broke things :)